src/core/Table/Table.tsx

```diff
@@ -322,31 +322,27 @@ export const Table = <
     <div
       ref={(element) => setOwnerDocument(element?.ownerDocument)}
       {...getTableProps({
-        className: cx('iui-tables-table', className),
+        className: cx('iui-table', className),
         style,
       })}
       {...ariaDataAttributes}
     >
       <div>
         {headerGroups.slice(1).map((headerGroup: HeaderGroup<T>) => {
           const headerGroupProps = headerGroup.getHeaderGroupProps({
-            className: 'iui-tables-row',
+            className: cx('iui-table-header', 'iui-row'),
```

Comment:
In HTML example this is the structure:
![image](https://user-images.githubusercontent.com/36186912/121854618-12d8fc80-ccfb-11eb-833d-a98b3e49141b.png)
Row should be a new element.

Reply:
Wow, I'm surprised it even worked before

Reply:
Oh it does nothing 😄 
![image](https://user-images.githubusercontent.com/9084735/121896061-e28b6100-ccee-11eb-8617-cac8a3551506.png)

because react-table sets flex automatically
![image](https://user-images.githubusercontent.com/9084735/121896107-ef0fb980-ccee-11eb-919c-81d6dc5728c5.png)